Modules/System/SharePoint/src/SharePointEvents.Codeunit.al

```diff
@@ -0,0 +1,12 @@
+codeunit 9112 "SharePoint Events"
+{
+    [IntegrationEvent(false, false)]
+    procedure ProcessSharePointFileMetadata(Metadata: JsonToken; SharePointFile: Record "SharePoint File" temporary)
+    begin
+    end;
+
+    [IntegrationEvent(false, false)]
+    procedure ProcessSharePointListItemMetadata(Metadata: JsonToken; SharePointListItem: Record "SharePoint List Item" temporary)
```

Comment:
Could you please add some documentation.
And shouldn't the SharePointListItem be a var parameter?